src/PioneerUtils.jsm

```diff
@@ -61,6 +66,7 @@ export class PioneerUtils {
   constructor(config) {
     this.config = config;
     this.encrypter = null;
+    this._logger = null;
```

Comment:
i guess the other two variables above this should also be made *"private"* for consistency but I'll file another PR with that later.

Reply:
This is actually mostly to distinguish it from the getter below, so you don't accidentally use the uninitialized version.